x-pack/solutions/security/plugins/security_solution/public/onboarding/components/onboarding_body/cards/integrations/integration_card_grid_tabs.tsx

```diff
@@ -78,6 +87,34 @@ export const IntegrationsCardGridTabsComponent = React.memo<IntegrationsCardGrid
 
     const selectedTab = useMemo(() => INTEGRATION_TABS_BY_ID[toggleIdSelected], [toggleIdSelected]);
 
+    const buttonGroupStyles = useMemo(
+      () => css`
+        button {
+          position: relative;
+        }
+        button:not(:first-child)::before {
+          position: absolute;
+          left: 0px;
+          content: ' ';
+          width: 1px;
+          height: 70%;
+          background-color: ${euiTheme.colors.darkShade};
+          top: 50%;
+          transform: translateY(-50%);
+        }
+        button[aria-pressed='true'],
+        button[aria-pressed='true'] + button {
+          &::before {
+            content: none;
+          }
+        }
+        button[aria-pressed='true'] {
+          text-decoration: none;
+        }
+      `,
+      [euiTheme.colors.darkShade]
+    );
+
```

Comment:
nit: this occupies a pretty big part of render body, can we move this out into a `useStyles` function?